api/src/main/java/io/opentelemetry/trace/Status.java

```diff
@@ -44,163 +43,18 @@
-
-    /**
-     * Operation was rejected because the system is not in a state required for the operation's
-     * execution. For example, directory to be deleted may be non-empty, an rmdir operation is
-     * applied to a non-directory, etc.
-     *
-     * <p>A litmus test that may help a service implementor in deciding between FAILED_PRECONDITION,
-     * ABORTED, and UNAVAILABLE: (a) Use UNAVAILABLE if the client can retry just the failing call.
-     * (b) Use ABORTED if the client should retry at a higher-level (e.g., restarting a
-     * read-modify-write sequence). (c) Use FAILED_PRECONDITION if the client should not retry until
-     * the system state has been explicitly fixed. E.g., if an "rmdir" fails because the directory
-     * is non-empty, FAILED_PRECONDITION should be returned since the client should not retry unless
-     * they have first fixed up the directory by deleting files from it.
-     *
-     * @since 0.1.0
-     */
-    FAILED_PRECONDITION(9),
-
-    /**
-     * The operation was aborted, typically due to a concurrency issue like sequencer check
-     * failures, transaction aborts, etc.
-     *
-     * <p>See litmus test above for deciding between FAILED_PRECONDITION, ABORTED, and UNAVAILABLE.
-     *
-     * @since 0.1.0
-     */
-    ABORTED(10),
 
     /**
-     * Operation was attempted past the valid range. E.g., seeking or reading past end of file.
-     *
-     * <p>Unlike INVALID_ARGUMENT, this error indicates a problem that may be fixed if the system
-     * state changes. For example, a 32-bit file system will generate INVALID_ARGUMENT if asked to
-     * read at an offset that is not in the range [0,2^32-1], but it will generate OUT_OF_RANGE if
-     * asked to read from an offset past the current file size.
-     *
-     * <p>There is a fair bit of overlap between FAILED_PRECONDITION and OUT_OF_RANGE. We recommend
-     * using OUT_OF_RANGE (the more specific error) when it applies so that callers who are
-     * iterating through a space can easily look for an OUT_OF_RANGE error to detect when they are
-     * done.
-     *
-     * @since 0.1.0
+     * The operation has been validated by an Application developers or Operator to have completed
+     * successfully.
      */
-    OUT_OF_RANGE(11),
-
-    /**
-     * Operation is not implemented or not supported/enabled in this service.
-     *
-     * @since 0.1.0
-     */
-    UNIMPLEMENTED(12),
-
-    /**
-     * Internal errors. Means some invariants expected by underlying system has been broken. If you
-     * see one of these errors, something is very broken.
-     *
-     * @since 0.1.0
-     */
-    INTERNAL(13),
-
-    /**
-     * The service is currently unavailable. This is a most likely a transient condition and may be
-     * corrected by retrying with a backoff.
-     *
-     * <p>See litmus test above for deciding between FAILED_PRECONDITION, ABORTED, and UNAVAILABLE.
-     *
-     * @since 0.1.0
-     */
-    UNAVAILABLE(14),
+    OK(0),
```

Comment:
Do we need the numbers anymore?

Reply:
I'm honestly not sure. I noticed that in the Jaeger mapping, we directly just copy that code over as a status code. Is that correct? No idea, either. I'd prefer to get rid of the numeric codes, and the accessor for them, but we need to decide what to do with the existing mappings.

Reply:
Got it - we can change this latter anyways.